backend/src/main/java/de/njsm/versusvirus/backend/service/purchase/PurchaseService.java

```diff
@@ -141,6 +141,27 @@ public void publishPurchase(UUID purchaseId) {
         messageSender.broadcastPurchase(customer, purchase);
     }
 
+    public void withdrawPurchase(UUID purchaseId) {
+        var purchase = purchaseRepository.findByUuid(purchaseId).orElseThrow(NotFoundException::new);
+        if (purchase.getAssignedVolunteer().isPresent()) {
+            throw new IllegalStateException("Cannot delete purchase after volunteer is assigned");
```

Comment:
Exception text talks about delete in withdraw context

Reply:
Alright, I fixed it :)